### PR TITLE
Lifecycle batch 2

### PR DESCRIPTION
**Pull request by alexduf**
This PR is the second part, here [is part one](https://github.com/guardian/frontend/pull/13115)

### CODE REVIEW

**Comment by alexduf** [comment]
Getting there!

File: sport/app/football/conf/context.scala [hunk 1]
```diff
@@ -5,15 +5,18 @@ import feed.Competitions
...
-trait FootballLifecycle extends GlobalSettings with ExecutionContexts {
+class FootballLifecycle(appLifeCycle: ApplicationLifecycle)(implicit ec: ExecutionContext) extends LifecycleComponent {
+
+  appLifeCycle.addStopHook { () => Future {
+    descheduleJobs()
+  }}
 
   private def scheduleJobs() {
-    Competitions.competitionIds.zipWithIndex map { case (id, index) =>
+    Competitions.competitionIds.zipWithIndex foreach { case (id, index) =>
```
Comment (TBonnin) [inline sport/app/football/conf/context.scala]: 😀

**Comment by TBonnin** [comment]
lgtm. Is dev-build compiling? ;)

**Comment by alexduf** [comment]
dev-build is compiling and running :)

**Pull request merged**